esp/esp/program/modules/base.py

```diff
@@ -292,23 +292,6 @@ def getTemplate(self):
         if self.module.inline_template:
             baseDir = 'program/modules/'+self.__class__.__name__.lower()+'/'
```

Comment:
I think you can even remove these two lines too now.  (Maybe some day we will turn on unused vars in the linter and it will catch these things!)